docs: update tutorial wrapping up (#338)
carbon-design-system/carbon-website: src/pages/tutorial/vue/wrapping-up.mdx

  ['Overview', 'Step 1', 'Step 2', 'Step 3', 'Step 4', 'Step 5', 'Wrapping up']
---

> ### Thanks for completing our tutorial and helping us improve it along the way. Want to show the world your new skills? Please apply for the IBM Digital Badge.

Reviewer: ```suggestion
### Thanks for completing our tutorial and helping us improve it along the way. If you want to show the world your new skills, you can apply for an IBM Digital Badge.```

<AnchorLinks>

<AnchorLink>Application form</AnchorLink>
<AnchorLink>Badge information</AnchorLink>

</AnchorLinks>

## Application form

> Once you complete all five steps of the tutorial, you can [apply for the IBM Digital Badge](https://www.surveygizmo.com/s3/5215482/IBM-Carbon-Design-System-Developer-Essentials-Vue-v2). These badges are available to anybody — not just IBM employees.

Reviewer: ```suggestion
Once you complete all five steps of the tutorial, you can apply for an [IBM Digital Badge](https://www.surveygizmo.com/s3/5215482/IBM-Carbon-Design-System-Developer-Essentials-Vue-v2). These badges are available to anybody — not just IBM employees.
```

</Column>
</Row>

> Once you submit the application, please allow 2-4 days for issuing the badge. Once the badge is issued, you will be notified and must accept the badge via Acclaim.

Reviewer: ```suggestion
Once you submit the application, please allow two to four days for issuing the badge. Once the badge is issued, you will be notified and must accept the badge via Acclaim.
```

NOTICE: IBM leverages the services of Credly's Acclaim platform, a 3rd party data processor authorized by IBM and located in the United States, to assist in the administration of the IBM Digital Badge program. In order to issue you an IBM Digital Badge, your personal information (name, email address, and badge earned) will be shared with the Credly's Acclaim platform. You will receive an email notification from Acclaim with instructions for claiming the badge. Your personal information is used to issue your badge and for program reporting and operational purposes. It will be handled in a manner consistent with IBM privacy practices. The IBM Privacy Statement can be viewed here: [https://www.ibm.com/privacy/us/en/](https://www.ibm.com/privacy/us/en/). IBM employees can view the IBM Internal Privacy Statement here: [https://w3.ibm.com/w3publisher/w3-privacy-notice](https://w3.ibm.com/w3publisher/w3-privacy-notice).

### Application form

> To apply for the IBM Digital Badge, please use [this link](https://www.surveygizmo.com/s3/5215482/IBM-Carbon-Design-System-Developer-Essentials-Vue-v2).

Reviewer: ```suggestion
To apply for the IBM Digital Badge, please complete the [form and survey](https://www.surveygizmo.com/s3/5215482/IBM-Carbon-Design-System-Developer-Essentials-Vue-v2).
```
